scalableminds/webknossos
# Splits dataset gallery and advanced dataset in separate tabs

### Mailable description of changes:
 - Splits dataset gallery and advanced dataset in separate tabs

### Steps to test:
- Open Dashboard
- Look at the two dataset tabs
- Open `Datasets` tab and reload; see that you're again at `Datasets` tab

------
